src/convertToObject.js

```diff
@@ -12,7 +12,16 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesArr = sourceString.trim().split(';');
+  stylesArr.pop();
+  const stylesObj = {};
+
+  stylesArr.forEach((property) => {
+    const [key, value] = property.split(':');
```

Comment:
```suggestion
    const [key, value] = property.trim().split(':');
```
you can write `trim` here